// vitepress-demo-preview-component/vite.config.ts.timestamp-1748079873534.mjs
// vite.config.ts
import { defineConfig } from "file:///E:/Projects/pla-element/node_modules/.pnpm/vite@3.2.11_@types+node@20.11.20_sass@1.85.1_terser@5.39.0/node_modules/vite/dist/node/index.js";
import { resolve } from "path";
import vue from "file:///E:/Projects/pla-element/node_modules/.pnpm/@vitejs+plugin-vue@3.2.0_vite@3.2.11_@types+node@20.11.20_sass@1.85.1_terser@5.39.0__vue@3.5.13_typescript@5.8.2_/node_modules/@vitejs/plugin-vue/dist/index.mjs";
import dts from "file:///E:/Projects/pla-element/node_modules/.pnpm/vite-plugin-dts@1.7.3_@types+node@20.11.20_rollup@4.34.9_vite@3.2.11_@types+node@20.11.20_sass@1.85.1_terser@5.39.0_/node_modules/vite-plugin-dts/dist/index.mjs";
var __vite_injected_original_dirname = "E:\\Projects\\pla-element\\_node_modules\\vitepress-demo-preview-component";
var vite_config_default = defineConfig({
  build: {
    outDir: "_product",
    lib: {
      entry: resolve(__vite_injected_original_dirname, "./index.ts"),
      name: "previewComponent",
      fileName: "preview-component"
    },
    rollupOptions: {
      external: ["vue"],
      output: {
        globals: {
          vue: "Vue"
        }
      }
    }
  },
  plugins: [vue(), dts({ insertTypesEntry: true })]
});
export {
  vite_config_default as default
};
